File: 100 Days Coding Part-1/5. Even Odd/evenodd.java
//               Day 5 Coding Statement

// Write a Java program to identify if the number is even or odd

// Description
//    → Get a number as input from the user and check whether the given number is odd or even

// •Input
//    12
// •Output
//    Even

// •Input
//   15
// •Output
//   Odd

import java.util.*;
public class evenodd {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter any Number: ");
        int n = sc.nextInt();

       if(n % 2 == 0)
        System.out.println(n + " is Even.");

       else
        System.out.println(n + " is Odd.");
    }
}

File: 100 Days Coding Part-1/5. Even Odd/evenodd.py
#               Day 5 Coding Statement

# Write a Python program to identify if the number is even or odd

# Description
#    → Get a number as input from the user and check whether the given number is odd or even

# •Input
#    12
# •Output
#    Even

# •Input
#   15
# •Output
#   Odd

n = int(input("Enter a Number: "))  #take input from user
if(n % 2 == 0):               # if remainder is zero then even
    print(n , " is Even." )
else:                         # if remainder is 1 then odd
    print(n," is Odd") 